src/player/SurvivalBlockBreakHandler.php

```diff
@@ -65,8 +67,27 @@ private function calculateBreakProgressPerTick() : float{
 		if(!$this->block->getBreakInfo()->isBreakable()){
 			return 0.0;
 		}
-		//TODO: improve this to take stuff like swimming, ladders, enchanted tools into account, fix wrong tool break time calculations for bad tools (pmmp/PocketMine-MP#211)
-		$breakTimePerTick = $this->block->getBreakInfo()->getBreakTime($this->player->getInventory()->getItemInHand()) * 20;
+		$breakTimePerSecond = $this->block->getBreakInfo()->getBreakTime($this->player->getInventory()->getItemInHand());
```

Comment:
This should really be `breakTimeSeconds`, "per second" doesn't make sense. (Don't ask me why the original variable was named that way ...)